podman.spec.rpkg

```diff
@@ -171,18 +170,23 @@ CGO_CFLAGS=$(echo $CGO_CFLAGS | sed 's/-specs=\/usr\/lib\/rpm\/redhat\/redhat-an
 export CGO_CFLAGS+=" -m64 -mtune=generic -fcf-protection=full"
 %endif
 
+# exclude_graphdriver_devicemapper doesn't work anymore with golang v1.19. Get
+# rid of this file instead.
+rm -rf vendor/github.com/containers/storage/drivers/register/register_devicemapper.go
```

Comment:
nit: Should this include if `go version` >= `v.1.19` check

Reply:
Is there an upstream issue for tracking?

Reply:
Probably worth referencing the issue with a `FIXME: #issue`

Reply:
looks like it's actually v1.18 which is on fedora 35 while the rhel envs in copr are on v1.17. I'll update the spec file.

Reply:
also, buildah rpm builds seem to be behaving differently, which I'm looking into right now. Holding this until there's consistency.

Reply:
> Probably worth referencing the issue with a `FIXME: #issue`

gotta check, that'd be worth adding, thanks!

Reply:
I found an inconsistent use of the new `go:build` tag format, https://github.com/containers/storage/pull/1417
However I don't think this should be an issue, go should be backwards compatible and still accept the old tag format if it only uses this in the file but worth to try again with that.